cookbooks/aws-parallelcluster-awsbatch/test/controls/awsbatch_config_spec.rb

```diff
@@ -9,7 +9,7 @@
 # This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
 # See the License for the specific language governing permissions and limitations under the License.
 
-control 'tag:config_awsbatch_correctly_configured' do
+control 'tag:config_awsbatch' do
```

Comment:
Maybe it's worth mentioning what we are checking here (for instance, `awsbatch_cli_commands_available`)